keyboards/flashquark/horizon_z/info.json

```diff
@@ -0,0 +1,166 @@
+{
+    "manufacturer": "FLASHQUARK",
+    "keyboard_name": "Horizon_Z",
+	"url": "https://flashquark.com/product/flashquark-horizon-z-qmk-edition-custom-kit/",
+    "maintainer": "FLASHQUARK",
+    "bootloader": "atmel-dfu",
+    "debounce": 3,
+    "diode_direction": "COL2ROW",
+    "processor": "atmega32u4",
+    "rgb_matrix": {
+        "driver": "IS31FL3733"
+		"layout": [
+			{"x":   0, "y":  0, "flags": 1},
+			{"x":   0, "y":  0, "flags": 4 }, 
+			{"x":  16, "y":  0, "flags": 4 }, 
+			{"x":  32, "y":  0, "flags": 4 }, 
+			{"x":  48, "y":  0, "flags": 4 }, 
+			{"x":  64, "y":  0, "flags": 4 }, 
+			{"x":  80, "y":  0, "flags": 4 }, 
+			{"x":  96, "y":  0, "flags": 4 }, 
+			{"x": 112, "y":  0, "flags": 4 },  
+			{"x": 128, "y":  0, "flags": 4 },  
+			{"x": 144, "y":  0, "flags": 4 }, 
+			{"x": 160, "y":  0, "flags": 4 }, 
+			{"x": 176, "y":  0, "flags": 4 }, 
+			{"x": 192, "y":  0, "flags": 4 }, 
+			{"x": 216, "y":  0, "flags": 1 },
+			{"x":   4, "y": 16, "flags": 4 }, 
+			{"x":  24, "y": 16, "flags": 4 }, 
+			{"x":  40, "y": 16, "flags": 4 }, 
+			{"x":  56, "y": 16, "flags": 4 }, 
+			{"x":  72, "y": 16, "flags": 4 }, 
+			{"x":  88, "y": 16, "flags": 4 }, 
+			{"x": 104, "y": 16, "flags": 4 }, 
+			{"x": 120, "y": 16, "flags": 4 },  
+			{"x": 136, "y": 16, "flags": 4 },  
+			{"x": 152, "y": 16, "flags": 4 }, 
+			{"x": 168, "y": 16, "flags": 4 }, 
+			{"x": 184, "y": 16, "flags": 4 }, 
+			{"x": 200, "y": 16, "flags": 4 }, 
+			{"x": 220, "y": 16, "flags": 4 },
+			{"x":   6, "y": 32, "flags": 4 }, 
+			{"x":  28, "y": 32, "flags": 4 }, 
+			{"x":  44, "y": 32, "flags": 4 }, 
+			{"x":  60, "y": 32, "flags": 4 }, 
+			{"x":  76, "y": 32, "flags": 4 }, 
+			{"x":  92, "y": 32, "flags": 4 }, 
+			{"x": 108, "y": 32, "flags": 4 }, 
+			{"x": 124, "y": 32, "flags": 4 },  
+			{"x": 140, "y": 32, "flags": 4 },  
+			{"x": 156, "y": 32, "flags": 4 }, 
+			{"x": 172, "y": 32, "flags": 4 }, 
+			{"x": 188, "y": 32, "flags": 4 }, 
+			{"x": 204, "y": 32, "flags": 1 },
+			{"x":  10, "y": 48, "flags": 1 }, 
+			{"x":  20, "y": 48, "flags": 4 }, 
+			{"x":  52, "y": 48, "flags": 4 }, 
+			{"x":  68, "y": 48, "flags": 4 }, 
+			{"x":  84, "y": 48, "flags": 4 }, 
+			{"x": 100, "y": 48, "flags": 4 }, 
+			{"x": 116, "y": 48, "flags": 4 }, 
+			{"x": 132, "y": 48, "flags": 4 },  
+			{"x": 148, "y": 48, "flags": 4 },  
+			{"x": 164, "y": 48, "flags": 4 }, 
+			{"x": 180, "y": 48, "flags": 4 }, 
+			{"x": 210, "y": 48, "flags": 1 },
+			{"x":   2, "y": 64, "flags": 1 }, 
+			{"x":  22, "y": 64, "flags": 1 }, 
+			{"x":  42, "y": 64, "flags": 1 }, 
+			{"x": 102, "y": 64, "flags": 1 }, 
+			{"x": 162, "y": 64, "flags": 1 }, 
+			{"x": 182, "y": 64, "flags": 1 }, 
+			{"x": 202, "y": 64, "flags": 1 }, 
+			{"x": 222, "y": 64, "flags": 1 }
```

Comment:
`"matrix": [row, col]` fields should be added for various animations that depend on them to work correctly.